pkg/daemon/controller/controller_test.go

```diff
@@ -134,19 +135,63 @@ func TestResolveNodeVariables(t *testing.T) {
 
 func TestResolvePowershellVariables(t *testing.T) {
 	testIO := []struct {
-		name      string
-		service   servicescm.Service
-		expected  map[string]string
-		expectErr bool
+		name            string
+		nodeName        string
+		nodeAnnotations map[string]string
+		nodeLabels      map[string]string
+		service         servicescm.Service
+		expected        map[string]string
+		expectErr       bool
 	}{
 		{
-			name:      "No Powershell variables to replace",
-			service:   servicescm.Service{},
+			name:            "No Powershell variables to replace",
+			nodeName:        "node",
+			nodeAnnotations: map[string]string{"desiredkey": "desiredvalue"},
+			nodeLabels:      map[string]string{"label": "labelvalue"},
+			service:         servicescm.Service{},
+			expected:        map[string]string{},
+			expectErr:       false,
+		},
+		{
+			name:            "Resolve powershell path variables",
+			nodeName:        "node",
+			nodeAnnotations: map[string]string{"desiredkey": "desiredvalue"},
+			nodeLabels:      map[string]string{"label": "labelvalue"},
+			service: servicescm.Service{
+				PowershellPreScripts: []servicescm.PowershellPreScript{{
```

Comment:
as is, this test is not doing anything.

In order to properly test that the replacements are occurring as expected, you need to add a `VariableName` and then add its entry to the test case's `expected` map